fibonacci/swift/code.swift

```diff
@@ -0,0 +1,19 @@
+@main
+struct Main {
+    static func fibonacci(_ n: Int) -> Int {
+        if n == 0 { return 0 }
+        if n == 1 { return 1 }
```

Comment:
this seems to be faster for me:
```swift
if n <= 1 { return n }
```

Reply:
But thats different than the other progs, so will leave as is.